src/styles/blocks/card.scss

```diff
@@ -0,0 +1,13 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
```

Comment:
it's better to use variables for all the colors and "magic numbers" 
https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/